src/main/webapp/partials/instructorHelpSections.jsp

```diff
@@ -0,0 +1,65 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="sections">Sections</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="sectionsWhatIsItMeantFor">
+                    <h3>1. What are sections meant for?</h3>
+                </a>
+            </span>
+            <p>
+                Sections are meant for organizing students in courses with significantly large numbers of students. It is mandatory for courses with more than 100 students to organize a student into a section.
+                You are allowed to have sections for courses with less than 100 students, but it is not mandatory.
+            </p>
+        </li>
+        <li>
+            <span class="text-bold">
+                <a name="sectionsEnrollingStudents">
+                    <h3>2. Enrolling students into sections</h3>
+                </a>
+            </span>
+            <p>
+                You can specify the section of each student at the time of enrollment. You just have to include a
+                <b>'Section'</b> column in the spreadsheet and ensure it is copied over to the student data text box, together with the rest of the data.
+                <br>More information can be found on the course enrollment page.
```

Comment:
By `Entire paragraph:` I meant you should delete this line (sorry wasn't clear enough!) Reason: no need to repeat `more information`; can be more concise.